src/spikeinterface/core/generate.py

```diff
@@ -1431,5 +1433,7 @@ def generate_ground_truth_recording(
     )
     recording.annotate(is_filtered=True)
     recording.set_probe(probe, in_place=True)
+    recording.set_property("gain_to_uV", np.ones(num_channels))
```

Comment:
Don't we have set_channel_gains for this?

Reply:
Yep I forgot.
I do it.

Reply:
done